webapp/src/main/java/org/apache/atlas/web/util/CachedBodyHttpServletRequest.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.atlas.web.util;
+
+import org.springframework.util.StreamUtils;
+
+import javax.servlet.ServletInputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletRequestWrapper;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class CachedBodyHttpServletRequest extends HttpServletRequestWrapper {
```

Comment:
Can we use this instead? 
![Uploading image.png…]()

Reply:
Its not posible for body with Application/JSON, only for Form Content
![image](https://user-images.githubusercontent.com/59254445/223034669-1160011e-ff60-4e91-a37a-a079515d278e.png)